elements/pfe-sass/maps/_general.scss

```diff
@@ -44,10 +44,10 @@ $pfe-vars: (
   grid-breakpoint--lg:           992px,
   grid-breakpoint--xl:           1200px,
 
-  box-shadow--sm:                '0  #{pfe-size-prem(1)}  #{pfe-size-prem(2)} 0 rgba(#{$pfe-color--gray-1000}, .2)',
-  box-shadow--md:                '0  #{pfe-size-prem(2)}  #{pfe-size-prem(1)} #{pfe-size-prem(1)} rgba(#{$pfe-color--gray-1000}, .12), 0  #{pfe-size-prem(4)}  #{pfe-size-prem(11)} #{pfe-size-prem(6)} rgba(#{$pfe-color--gray-1000}, .05)',
-  box-shadow--lg:                '0  #{pfe-size-prem(3)}  #{pfe-size-prem(7)} #{pfe-size-prem(3)} rgba(#{$pfe-color--gray-1000}, .13), 0  #{pfe-size-prem(11)}  #{pfe-size-prem(24)} #{pfe-size-prem(16)} rgba(#{$pfe-color--gray-1000}, .12)',
-  box-shadow--inset:             'inset 0 0 #{pfe-size-prem(10)} 0 #{$pfe-color--gray-50}',
+  box-shadow--sm:                #{0  #{pfe-size-prem(1)}  #{pfe-size-prem(2)} 0 rgba($pfe-color--gray-1000, .2)},
```

Comment:
These didn't resolve because of the quote marks and the fact that the functions weren't being loaded until after the maps.